Documentation/ColumnsConfig/Type/Inline/Properties/SymmetricField.rst

```diff
@@ -14,3 +14,10 @@ symmetric\_field
    This works like :ref:`foreign_field <columns-inline-properties-foreign-field>`, but in case of using
    bidirectional symmetric relations. :code:`symmetric_field` defines in which field on
    the :ref:`foreign_table <columns-inline-properties-foreign-table>` the uid of the "other" parent is stored.
+
+Example
+=======
+
+.. include:: /CodeSnippets/InlineMnSymetricInline1.rst.txt
```

Comment:
Symmetric is spelled with two m (see text block before this) - we should probably change that at wherever the source of these is.